### ISSUE TITLE
CLion Autocomplete not detecting some pub items from imported crates

### ISSUE THREAD

**Issue by rsdlt** [intellij-rust/intellij-rust#10226]
## Environment

* **IntelliJ Rust plugin version:** 0.4.189.5234-223
* **Rust toolchain version:** 1.68.0 (2c8cc3432 2023-03-06) x86_64-pc-windows-msvc
* **IDE name and version:** CLion 2022.3.2 (CL-223.8617.54)
* **Operating system:** Windows 11 10.0
* **Macro expansion:** enabled

## Problem description
CLion autocomplete is not showing or detecting some items from imported crates nor applying color schemes / inlay hints, while correctly detecting other items from the same crate. Cargo check and Clippy correctly detect problems with imported items, even if not detected by autocomplete. 

## Steps to reproduce

1. Add 'ash' crate in Cargo.toml:

```toml
[dependencies]
ash = { version = "0.37.2"}
```
2. Import 'ash' crate in main.rs:
```rust
use ash::{self,vk};
```
3. Use imported modules within `main()` function in `main.rs`, for example:
- `vk::InstanceCreateFlags` is detected by autocomplete, inlay hints are displayed and color scheme applied.
- `vk::InstanceCreateInfo` is not detected by autocomplete, inlay hints not displayed, color scheme not applied, cargo check and clippy correctly identify no problems.
- `vk::InstanceCreateInfoBuilder` is not detected, inlay hints not displayed, color scheme not applied, cargo check and clippy correctly identify problems.
- `vk::ApplicationInfo` is not detected, inlay hints not displayed, color scheme not applied, cargo check and clippy correctly identify no problems.

The three  `structs` are `pub` and are properly detected by the autocomplete in other IDEs.

![image](https://user-images.githubusercontent.com/109830882/224371567-98e57570-1e28-494d-a415-71856ef8b67c.png)

![image](https://user-images.githubusercontent.com/109830882/224375411-947fe2f9-d428-4df2-beda-fa8ef26427fd.png)

Crate documentation: https://docs.rs/ash/0.37.2+1.3.238/ash/vk/index.html

Full code:
`Cargo.toml`:
```toml
[package]
name = "ash-play"
version = "0.1.0"
edition = "2021"

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dependencies]
anyhow = "1.0.69"
ash = { version = "0.37.2"}
```

`main.rs`:
```rust
#![allow(unused)]

use anyhow::Result;
use ash::{self, vk};

fn main() -> Result<()> {
    // Load default Vk library:
    // Required to create a VK instance.
    let entry = unsafe { ash::Entry::load() }?;

    let create_flag = vk::InstanceCreateFlags::empty();
    let create_info = vk::InstanceCreateInfo::builder();
    let create_info_b = vk::InstanceCreateInfoBuilder;
    let app_info = vk::ApplicationInfo::builder();

    // Create an Instance:
    // Initializes the Vulkan library and allows the application to pass information about itself
    // to the implementation.
    let instance = unsafe { entry.create_instance(&create_info, None) }?;
    Ok(())
}
```

Thanks in advance for your support.

**Comment by vlad20012** [comment]
The crate `ash` contains a file `src/vk/definitions.rs` which exceeds the code insight file size limit. 

![image](https://user-images.githubusercontent.com/3221931/225923147-8379bb0f-7ffb-499f-ad65-0b2a9658c58b.png)

The limit can be adjusted using `idea.max.intellisense.filesize` property. See [this docs](https://www.jetbrains.com/help/objc/configuring-file-size-limit.html) for instructions.


Related issue - #10197
